wiki/Twitter/fr.md

```diff
@@ -1,16 +1,13 @@
----
-outdated: true
----
-
 # Twitter
 
 ## Nous tweetons aussi!
 
-osu! a plusieurs comptes Twitter, chacun sert à quelque chose de différent. La plupart ne tweetent qu'un peu mais vous gardent au courant de ce qui se passe dans l'écosystème d'osu!
+osu! a plusieurs comptes Twitter, chacun servant des objectifs spécifiques. La plupart sont très peu fréquentés mais vous aideront à vous tenir informés en dehors de l'osu! écosystème.
 
 | Avatar | Compte | Description |
 | :-: | :-: | --- |
-| ![osu! avatar](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | Un flux des actualités et des notifications des [beatmaps récemment rankées](https://osu.ppy.sh/p/beatmaplist). |
-| ![osu!status avatar](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Dédié à fournir de rares notifications sur les problèmes du site et de bancho. |
-| ![Dean Herbert avatar](img/ppy.jpg) | [@ppy](https://twitter.com/ppy) | Pas strictement lié à osu!, mais je le laisse ici pour ceux qui sont intéressés. Ma vie c'est osu!, après tout. |
-| ![BanchoBoat avatar](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Un élan comique quand les choses se passent mal. |
+| ![osu! avatar](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | La source officielle de nouvelles et d'annonces. Contenu de haute qualité à faible trafic. |
```

Comment:
```suggestion
| ![osu! avatar](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | Un flux des actualités et des notifications des [beatmaps récemment rankées](https://osu.ppy.sh/p/beatmaplist) |
```

Reply:
if you want to change the content/meaning you should either change EN first or open an issue